.github/workflows/publish-docker.yml

```diff
@@ -19,8 +34,8 @@ on:
         description: "type of nm-vllm to install for the docker image: NIGHTLY (default) or RELEASE"
         type: string
         default: 'NIGHTLY'
-      build_version:
-        description: "version of nm-vllm to install for the docker image: latest (default) or specific version e.g. 0.4.0, 0.4.0.20240531"
+      build_wheel:
```

Comment:
it might be better for this to be named `whl_version`.

Reply:
I renamed it to wheel as we actually need the wheel full name.